src/api/v1/mock/mock.controller.ts

```diff
@@ -12,14 +12,14 @@ import { MockService } from './mock.service';
 export class MockController {
     constructor(private readonly mockService: MockService) {}
 
-    // 특정 원룸 조회
+    //특정원룸 조회
     @Get('rooms/:id')
     async findOneByRoomId(@Param('id', ParseIntPipe) id: number) {
         const room = await this.mockService.findOneByRoomId(id);
         return room;
```

Comment:
변수를 선언하고 처리하는 로직이 없다면, 불필요한 선언 없이 바로 반환하는 건 어떨까요
```suggestion
        return await this.mockService.findOneByRoomId(id);